kernel/Kconfig

```
@@ -1029,6 +1029,30 @@ config BOUNDS_CHECK_BYPASS_MITIGATION
 	  mitigations after bounds checking any array index parameters passed
 	  in from untrusted sources (user mode threads). When disabled, these
 	  macros do nothing.
+
+config HW_SHADOW_STACK
+	bool "Uses hardware provided mechanism to provide stack protection"
```

```suggestion
	bool "Use hardware shadow stack mechanism to provide stack protection [EXPERIMENTAL]"
```
since the original test basically copies what `CONFIG_HW_STACK_PROTECTION` has: `Select this option to enable hardware-based platform features to catch stack overflows when the system is running in privileged mode.`